test/SpokePoolClient.ValidateFill.ts

```diff
@@ -353,7 +359,7 @@ describe("SpokePoolClient: Fill Validation", function () {
     // Searching for deposit ID -1 that doesn't exist should throw.
     await assertPromiseError(
       getBlockRangeForDepositId(
-        -1,
+        toBN(-1),
```

Comment:
It seems like this was probably left out of the recent BigNumber depositId migration.